.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test
```

Comment:
Run tests before each commit

Reply:
Why don't yarn? And i think it will be annoying. Can we run tests at least before pushing?

Reply:
I agree, I didn't think/know this was an option :)